apps/showcase-e2e/src/integration/cypress-indexedDB-helper.spec.ts

```diff
@@ -1,87 +1,111 @@
 describe(`@this-dot/cypress-indexeddb`, () => {
   beforeEach(() => {
     cy.clearIndexedDb('FORM_CACHE');
-    cy.openIndexedDb('FORM_CACHE')
-      .as('formCacheDB')
-      .createObjectStore('keyvaluepairs')
-      .as('objectStore');
+    cy.openIndexedDb('FORM_CACHE').as('formCacheDB');
   });
 
-  it(`entering data into the form saves it to the indexedDb`, () => {
-    cy.visit('/cypress-helpers');
+  describe(`key-value pair based databases`, () => {
+    beforeEach(() => {
+      cy.getIndexedDb('@formCacheDB').createObjectStore('keyvaluepairs').as('objectStore');
+    });
 
-    cy.get('#firstName').should('be.visible').type('Hans');
-    cy.get('#lastName').should('be.visible').type('Gruber');
-    cy.get('#country').should('be.visible').type('Germany');
-    cy.get('#city').should('be.visible').type('Berlin');
+    it(`entering data into the form saves it to the indexedDb`, () => {
+      cy.visit('/cypress-helpers');
+
+      cy.get('#firstName').should('be.visible').type('Hans');
+      cy.get('#lastName').should('be.visible').type('Gruber');
+      cy.get('#country').should('be.visible').type('Germany');
+      cy.get('#city').should('be.visible').type('Berlin');
```

Comment:
Maybe it should be good to create a custom command for this to avoid duplication?

```ts
Cypress.Commands.add('setValue', (id, value) => {
  cy.get(id).should('be.visible').type(value);
})
```

And then use it as follows:

```ts
cy.setValue('#firstName', 'Hans');
cy.setValue('#lastName', 'Gruber');
```

Reply:
While I understand the idea behind this, and I quite agree with the principle, I decided not to create custom commands for these tests for the following reason:

These tests are the same as we have in the how-to blog post. So if somebody comes from there to check out these tests without much knowledge about cypress commands it would be confusing.

And also this is not that much of a duplication, and it is readable line-by-line.

Reply:
I'm ok with this reasoning. Let's keep it as it is for a better understanding of the readers and users.